components/IndicatorStat/utils/data.ts

```diff
@@ -104,45 +104,54 @@ const aggregateProcessedData = (data: ProcessedData[], options: ValueOptions): n
   return data;
 };
 
-const processMultipleData = (data: LocationData[], options: ValueOptions = { dataFormat: 'plain' }): string => {
+const processMultipleData = (
+  data: LocationData[],
+  options: ValueOptions = { dataFormat: 'plain' },
+  decimalCount: number
+): string => {
   const sortedData = data.sort((a, b) => a.year - b.year);
   const latest = sortedData.filter(d => d.year === sortedData[data.length - 1].year);
   if (latest && latest.length > 1) {
     const _data = getOneFromMultipleBudgetTypes(data);
     if (_data) {
-      return getValue(_data, options);
+      return getValue(_data, options, decimalCount);
     }
   }
 
   if (!options.aggregation) {
     if (sortedData[data.length - 1].value) {
-      return getValue(sortedData[data.length - 1], options);
+      return getValue(sortedData[data.length - 1], options, decimalCount);
     }
   } else {
     const aggregate = aggregateProcessedData(data.map(locationDataToProcessedData), options);
     if (typeof aggregate === 'number') {
-      return addPrefixAndSuffix(formatNumber(aggregate), options);
+      return addPrefixAndSuffix(formatNumber(aggregate, decimalCount), options);
     }
   }
 
   return DEFAULT_VALUE;
 };
 
-export const getIndicatorValue = (data?: LocationData[], options: ValueOptions = { dataFormat: 'plain' }): string => {
+export const getIndicatorValue = (
+  data?: LocationData[],
+  options: ValueOptions = { dataFormat: 'plain' },
+  decimalCount?: number
+): string => {
   if (data && data.length) {
     if (data.length === 1 && data[0].value) {
-      return getValue(data[0], options);
+      return getValue(data[0], options, typeof decimalCount == 'number' ? decimalCount : 1);
```

Comment:
`decimalCount => ValueOptions`
Also, decimalCount should only be a number, so I don't get the purpose of the `typeof` check.

Reply:
Updated ValueOptions to include decimalCount